Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -168,9 +171,10 @@ public struct PackageBuilder {
         if path.basename.hasPrefix(".") { return false }
         if path == manifest.path { return false }
         if excludedPaths.contains(path) { return false }
-        if !path.asString.isFile { return false }
-        guard let ext = path.asString.fileExt else { return false }
-        return validExtensions.contains(ext)
+        if !fileSystem.isFile(path) { return false }
+        guard let ext = path.suffix else { return false }
+        // FIXME: This isn't efficient.
+        return validExtensions.contains(String(ext.utf8.dropFirst()))
```

Comment:
There should be a method on `path` to get this (the `extension`, defined to be the `suffix` without the `.`).